pkgs/applications/graphics/djv/default.nix

```
@@ -150,6 +151,7 @@ stdenv.mkDerivation rec {
     freetype
     opencolorio_1
     djv-deps
+    openexr.dev
```

```suggestion
    openexr
```

(This is completely equivalent.)

---

(This also applies to zlib above.)

---

:+1: